code/datums/mutations/antenna.dm

```diff
@@ -51,6 +51,18 @@
 	difficulty = 8
 	locked = TRUE
 
+/datum/mutation/human/mindreader/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+
+	owner.add_traits(TRAIT_MIND_READER, GENETIC_MUTATION)
```

Comment:
Could (should?) grant this with the action rather than the mutation. 

Unless you explicitly want to tie this to the mutation instead of the action it grants for whatever reason